### PR TITLE
Drf

### PR DESCRIPTION
**Pull request by Baranovsky1986**
on one documentation site, I found that I need to change these four files in this way. could you see if it’s right

### CODE REVIEW

**Comment by criticus** [comment]
do you need to add all 186 files? I think you submitted psycopg2 by mistake

**Comment by Baranovsky1986** [comment]
I cloned the project again and it was cloned with psycopg2. how about modified files?

File: data_converter/wsgi.py [hunk 1]
```diff
@@ -11,6 +11,6 @@
 
 from django.core.wsgi import get_wsgi_application
 
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "data_converter.settings")
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "{{ data_converter }}.settings")
```
Comment (criticus) [inline data_converter/wsgi.py:14]: why do we need curly brackets {{}}?

File: manage.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 import sys
 
 if __name__ == "__main__":
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "data_converter.settings")
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "{{ data_converter }}.settings")
```
Comment (criticus) [inline manage.py:6]: same question here

**Comment by Baranovsky1986** [comment]
not relevant

**Pull request closed without merging**